cpp/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ project(arcticdb VERSION 0.0.1)
 
 enable_testing()
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Could just inspect the ninja/make files directly?

Reply:
For ninja you have compdb which can extract it, for make you have to install [python package](https://pypi.org/project/compiledb/). On the other hand this will work the same way regardless of the build system.

Reply:
```vim
Ctrl-C
ESC
Ctrl-Alt-Delete
```

Reply:
> Ctrl-C
ESC
Ctrl-Alt-Delete

I don't get this.

Reply:
Just a joke on how difficult it is to quit Vim :)